### PR TITLE
Add more bills to wallets, add bundle of cash crafting

### PR DESCRIPTION
**Pull request by natsirt721**
#### Summary
Content "Add two, five, fifty, and one-hundred dollar bills"

#### Purpose of change
Wallets currently only contain one (rarely), ten and twenty dollar bills, and should have more varied paper money. Additionally, managing large amounts of currency can lead to performance impacts - this is true for large amounts of any item, but money tends to be aggregated and is particularly prone to this.

Partially addresses #48903 

#### Describe the solution
Add two, five, fifty, and one-hundred dollar bills. Add a crafting recipe for the bundle of cash to allow 100 of any single bill type to be converted into a bundle of cash and back again. Bundles of cash created at worldgen will disassemble into twenties as the original description suggested. Tweak the wallet item groups to include the new bills, and adjust item spawn rates, but these are certainly open for interpretation.
Also added item inheritance to money, and added three new item groups for wallet contents.

#### Describe alternatives you've considered
Considered adding separate items for each denomination of bundle (e.g. "bundle of ones", "bundle of fives" etc). Adds a lot of extra crafting recipes and items, but allows bundles to organize by denomination. The one-bundle approach causes all bundles to stack together and they must be inspected individually to determine what denomination of bill there is, but it is also compatible with current saves.

Considered allowing short string/cord instead of paper for the binding material in the bundle crafting recipe. But by the time you have enough bills to start bundling them you probably have access to paper.

Considered requiring an adhesive component for the bundle crafting recipe. Using 1 duct tape would be most appropriate, but in the interest of keeping the recipe simple this was left out.

Considered adding a nonzero price to bills/bundles. FM currency is 'printed' on fifties and hundreds, so there is some demand for those bills, although given the size of the economy it doesn't seem like the demand would be high enough to fetch a meaningful price.

#### Testing
Tested the new wallet item groups using debug menu, and ran around with a shotgun pulping zombies until a few wallets dropped. Tweaked the cash rates based on those findings.

### CODE REVIEW

File: data/json/items/generic.json [hunk 1]
```diff
@@ -2445,48 +2445,75 @@
...
+  {
+    "type": "GENERIC",
+    "id": "money_fifty",
+    "name": { "str": "fifty dollar bill" },
+    "description": "A fifty dollar bill, pretty useless now though.",
+    "copy-from": "paper_money"
+  },
+  {
+    "type": "GENERIC",
+    "id": "money_benjamin",
```
Comment (LaVeyanFiend) [inline data/json/items/generic.json:2513]: Shouldn't this be money_hundred, to keep it consistent with money_[numerical amount]?

Comment (natsirt721) [reply data/json/items/generic.json]: yes

**Comment by actual-nh** [comment]
You might want to take a look at #48298 also.

File: data/json/itemgroups/Clothing_Gear/wallets.json [hunk 2]
```diff
@@ -30,208 +30,156 @@
...
+      { "item": "money_two", "prob": 1 },
+      { "item": "money_five", "prob": 80, "count": [ 1, 5 ] },
+      { "item": "money_ten", "prob": 60, "count": [ 1, 3 ] },
+      { "item": "money_twenty", "prob": 35, "count": [ 1, 3 ] },
+      { "item": "money_fifty", "prob": 5 },
+      { "item": "money_hundred", "prob": 3 },
+      { "item": "coin_quarter", "prob": 10, "count": [ 1, 3 ] },
+      { "item": "coin_nickel", "prob": 10, "count": [ 1, 3 ] },
+      { "group": "discount_cards", "prob": 10 },
+      { "item": "condom", "prob": 10 },,
```
Comment (actual-nh) [inline data/json/itemgroups/Clothing_Gear/wallets.json:54]:
```suggestion
      { "item": "condom", "prob": 10 },
```

**Comment by natsirt721** [comment]
> You might want to take a look at #48298 also.
Wow, that's awkward. I neglected to consider that there are 200+ open PRs on string/content freeze at the moment and assumed that this was a novel contribution. Closing this PR in favor of that one.

**Pull request closed without merging**